### PR TITLE
COM-12575: Investigate erratic behaviour of AAC audio reference decoders

### PR DESCRIPTION
**Pull request by rsanchez87**
* Include improvements in mpeg reference decoders building
* Add JSON generation from ISO-MPEG2-AAC decoder in "iso_mpeg2_aac.py" file
* Change ""result":" values of MPEG2_AAC-ADIF.json and MPEG2_AAC-ADTS.json, from reference decoder, .pcm files

### CODE REVIEW

File: fluster/decoders/iso_mpeg2_aac.py [hunk 1]
```diff
@@ -42,10 +43,23 @@ def decode(
...
+        pcm_out_f00_file = f"{base_output}_f00.pcm"
+
+        if os.path.exists(pcm_out_f00_file):
+            return file_checksum(pcm_out_f00_file)
+
+        output_files = [f"{base_output}_f{str(i).zfill(2)}.pcm" for i in range(20)]
+
+        for pcm_file in output_files:
+            if os.path.exists(pcm_file):
+                return file_checksum(pcm_file)
```
Comment (rgonzalezfluendo) [inline fluster/decoders/iso_mpeg2_aac.py:63]: [glob](https://docs.python.org/3/library/glob.html) module can help here.

Comment (rsanchez87) [reply fluster/decoders/iso_mpeg2_aac.py:63]: something like:  ??

```
output_files = glob.glob(f"{base_output}_f[0-9][0-9].pcm")

for pcm_file in output_files:
    if os.path.exists(pcm_file):
        return file_checksum(pcm_file)
```

Comment (rsanchez87) [reply fluster/decoders/iso_mpeg2_aac.py]: Done

**Review by mdimopoulos — APPROVED** [review]

File: fluster/decoders/iso_mpeg2_aac.py [hunk 2]
```diff
@@ -42,10 +44,23 @@ def decode(
...
+        base_output = output_filepath[:-4]
+        pcm_out_f00_file = f"{base_output}_f00.pcm"
+
+        if os.path.exists(pcm_out_f00_file):
+            return file_checksum(pcm_out_f00_file)
+
+        output_files = glob.glob(f"{base_output}_f[0-9][0-9].pcm")
+
+        for pcm_file in output_files:
+            if os.path.exists(pcm_file):
```
Comment (rgonzalezfluendo) [inline fluster/decoders/iso_mpeg2_aac.py:63]: `glob.glob` only return existing files.

**Review by rgonzalezfluendo — APPROVED** [review]

**Pull request merged**